### DISCUSSION TITLE
Añadir un alto máximo al elemento Divider

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1350] [Proposals]
### Goal
Evitar que al escalar un Divider se convierta en un bloque

### Context

A día de hoy al escalar el elemento divider a lo ancho, a menudo sin querer terminas conviertiendo el divider en un bloque gris.
<img width="448" alt="image" src="https://github.com/Telefonica/mistica-design/assets/95479574/d52a9f08-b62e-43a2-b508-4a5502675bc1">

### Impact

Facilitaría usar el componente Divider

### Discussion Questions

¿Esto se podría solucionar con la nueva feature de max height?

**Comment by aweell** [comment]
Confirmo que se puede solucionar con max-height, he abierto una issue #1355 para solucionar este problema.
